README.md

```diff
@@ -1,216 +1,281 @@
+                   )
+               }
+           }
+       }
+   }
+   ```
+
+2. Update the shared code by adding a text field that will update the name displayed on the button:
+
+   ```diff
+   @OptIn(ExperimentalResourceApi::class)
+   @Composable
+   fun App() {
+       MaterialTheme {
+           var greetingText by remember { mutableStateOf("Hello, World!") }
+           var showImage by remember { mutableStateOf(false) }
+           Column(Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
+               Button(onClick = {
+                   greetingText = "Hello, ${getPlatformName()}"
+                   showImage = !showImage
+               }) {
+                   Text(greetingText)
+               }
+   +           TextField(greetingText, onValueChange = { greetingText = it })
+               AnimatedVisibility(showImage) {
+                   Image(
+                       painterResource("compose-multiplatform.xml"),
+                       null
+                   )
+               }
+           }
+       }
+   }
+   ```
+
+3. Re-run both `androidApp` and `iosApp` configurations. You'll see this change reflected on both the Android and iOS apps:
+
+   <img src="readme_images/text_field_added.png" height="200px">
+
+## How to configure the iOS application
+
+To get a better understanding of this template's setup and learn how to configure the basic properties of your iOS app without Xcode,
+open the `iosApp/Configuration/Config.xcconfig` file in Android Studio. The configuration file contains:
+
+* `APP_NAME`, a target executable and an application bundle name
+* `BUNDLE_ID` that [uniquely identifies the app throughout the system](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleidentifier#discussion)
+* `TEAM_ID`, [a unique identifier generated by Apple that's assigned to your team](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect)
+
+To configure the `APP_NAME` option, open `Config.xcconfig` in any text editor *before opening* the project in Android Studio, and set the desired name.
+
+If you need to change this option after you open the project in Android Studio, do the following:
+
+1. Close the project in Android Studio.
+2. Run `./cleanup.sh` in your terminal.
+3. Change the setting.
+4. Open the project in Android Studio again.
+
+To configure advanced settings, use Xcode. After opening the project in Android Studio,
+go to Xcode and open the `iosApp/iosApp.xcworkspace` file, and make changes.
+
+## Next steps
+
+We encourage you to further explore Compose Multiplatform and try out more projects:
+
+* [Learn about other cases where you can use the Compose Multiplatform UI framework](https://github.com/JetBrains/compose-multiplatform)
+* [Complete more Compose Multiplatform tutorials](https://github.com/JetBrains/compose-multiplatform#tutorials)
```

Comment:
@igordmn wants to create an index of tutorials so that we could refer it. 
I would also add a link to Compose for Desktop template.